assets/js/lib/markers/triageMarker.js

```diff
@@ -0,0 +1,48 @@
+class TriageMarker {
+  constructor(location, sheet_id) {
+    this.location = location;
+    this.marker = new google.maps.Marker({
+      position: location.position,
+      map: null,
+      title: location.title,
+      icon: "assets/images/triage.png"
+    });
+    this.position = this.marker.position;
+
+    var nodal_contact_info = []
+
+    for (var idx = 0; idx < location.nodal_officer_contact.length; idx++) {
+      nodal_contact_info.push('<a href="tel:' + location.nodal_officer_contact[idx] + '">' + location.nodal_officer_contact[idx] + '</a>')
+    }
+
+    var nodal_contact_details = nodal_contact_info.join('<br/>')
```

Comment:
Instead of doing the looks above. You [`.map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) and [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). This will improve the readability a lot:

```js
    let nodalContactDetails = location.nodal_officer_contact.map((contactNumber) => {
      return `<a href="tel:${contactNumber}">${contactNumber}</a>`;
    }).join("<br/>");
```